arrow-fx-test/src/main/kotlin/arrow/fx/test/laws/ConcurrentLaws.kt

```diff
@@ -279,7 +279,7 @@ object ConcurrentLaws {
   fun <F> Concurrent<F>.cancellableReceivesCancelSignal(EQ: Eq<Kind<F, Int>>, ctx: CoroutineContext) =
     forAll(Gen.int()) { i ->
       fx.concurrent {
-        val release = Promise.uncancellable<F, Int>(this@cancellableReceivesCancelSignal).bind()
+        val release = Promise<F, Int>(this@cancellableReceivesCancelSignal).bind()
```

Comment:
This is just a nit, I don't think it'll affect anything but like this is equivalent to the cancellableF